### PR TITLE
Extract persisted into a shared library

### PR DESCRIPTION
**Pull request by Caleb-T-Owens**

### CODE REVIEW

File: apps/desktop/src/lib/components/Board.svelte [hunk 1]
```diff
@@ -65,14 +66,8 @@
...
 	const handleKeyDown = createKeybind({
-		's t a c k': () => {
+		's t a c k': async () => {
 			$stackingFeature = !$stackingFeature;
-		},
-		'$mod+Shift+H': () => {
-			$showHistoryView = !$showHistoryView;
-		},
-		'$mod+z': () => {
-			$showHistoryView = !$showHistoryView;
```
Comment (estib-vega) [inline apps/desktop/src/lib/components/Board.svelte]: Is this intentional? I think then change being un-done was introduced by @ndom91 in here: https://github.com/gitbutlerapp/gitbutler/pull/5094/files#diff-3a92322cb1896e0e3dd44e9b66fe4c2536081848ab1755f9b30a7c47f61c3e42R68-R75

File: apps/desktop/src/lib/stack/StackHeader.svelte [hunk 2]
```diff
@@ -122,7 +122,6 @@
 						<Button
 							bind:el={meatballButtonEl}
 							style="ghost"
-							size="tag"
```
Comment (estib-vega) [inline apps/desktop/src/lib/stack/StackHeader.svelte]: Also these little changes, are they intentional quick fixes? Just want to double check

File: apps/desktop/src/lib/stack/StackHeader.svelte [hunk 3]
```diff
@@ -205,7 +205,7 @@
 	.header__info-wrapper {
 		display: flex;
 		gap: 2px;
-		padding: 12px;
+		padding: 10px;
```
Comment (estib-vega) [inline apps/desktop/src/lib/stack/StackHeader.svelte]: See comment above ☝️

File: apps/desktop/src/lib/stack/StackHeader.svelte [hunk 4]
```diff
@@ -218,7 +218,7 @@
 	.button-group {
 		display: flex;
 		align-items: center;
-		gap: 6px;
+		gap: 10px;
```
Comment (estib-vega) [inline apps/desktop/src/lib/stack/StackHeader.svelte]: See comment above ☝️

File: apps/desktop/src/lib/components/Board.svelte (hunk 1 above)
Comment (ndom91) [reply apps/desktop/src/lib/components/Board.svelte]: Yeah seems like you might be missing some latest changes from master

**Comment by Caleb-T-Owens** [comment]
Interesting, I think vscode stores cached versions of files which can cause it to crap out when you rename too much stuff

**Review by estib-vega — APPROVED** [review]
Looks good!

**Pull request merged**